hw/formal/tools/jaspergold/parse-formal-report.py

```diff
@@ -205,7 +205,6 @@ def main():
                   len(err_msgs.get("unreachable")))
     if n_errors > 0 or n_failures > 0:
         log.info("Found %d errors,  %d failures", n_errors, n_failures)
-        return 1
 
```

Comment:
Here if we return 1, then dvsim will treat it as a build error, then the actual pass rate won't show up in the generated result. So here we return 0 because the formal regression is finished and result is parsed, the actual pass/fail rate can refer to the result summary.